### PR TITLE
Include Forest 🌲 snapshot links

### PR DESCRIPTION
**Pull request by lerajk**
This PR contains snapshot links to be included from the Forest 🌲 node (maintained by ChainSafe).

### CODE REVIEW

**Review by smagdali — CHANGES_REQUESTED** [review]
Do you want to add some explanation of the difference between the two, and make 'Forest by Chainsafe" a link to the project home page, or more info? 

(but thanks for fixing the 'snapsnot' typo :) )

**Review by smagdali — APPROVED** [review]

File: content/en/networks/calibration/details/index.md [hunk 1]
```diff
@@ -56,3 +56,4 @@ Calibration network is the most realistic simulation of the Filecoin mainnet.
 - [Stats Dashboard](https://stats.calibration.fildev.network/)
 - [Slack Channel for Updates: #fil-network-announcements](https://filecoinproject.slack.com/archives/C01AC6999KQ)
 - [Slack Channel for Questions: #fil-help](https://filecoinproject.slack.com/archives/CEGN061C5)
+- [Latest lightweight snapshot](https://forest.chainsafe.io/calibnet/snapshot-latest) generated with [Forest 🌲](http://github.com/ChainSafe/forest) by [ChainSafe](https://chainsafe.io/)
```
Comment (johnnymatthews) [inline content/en/networks/calibration/details/index.md:59]:
```suggestion
- [Latest lightweight snapshot](https://forest.chainsafe.io/calibnet/snapshot-latest) generated with [Forest](http://github.com/ChainSafe/forest) by [ChainSafe](https://chainsafe.io/)
```

File: content/en/networks/mainnet/details/index.md [hunk 2]
```diff
@@ -60,7 +60,8 @@ aliases:
 
 ## Resources
 
-- [Latest lightweight snapsnot](https://snapshots.mainnet.filops.net/minimal/latest)
+- [Latest lightweight snapshot](https://snapshots.mainnet.filops.net/minimal/latest) generated with [Lotus](https://lotus.filecoin.io/) by [Protocol Labs](https://protocol.ai/)
+- [Latest lightweight snapshot](https://forest.chainsafe.io/mainnet/snapshot-latest) generated with [Forest 🌲](http://github.com/ChainSafe/forest) by [ChainSafe](https://chainsafe.io/)
```
Comment (johnnymatthews) [inline content/en/networks/mainnet/details/index.md:64]:
```suggestion
- [Latest lightweight snapshot](https://forest.chainsafe.io/mainnet/snapshot-latest) generated with [Forest](http://github.com/ChainSafe/forest) by [ChainSafe](https://chainsafe.io/)
```

**Review by johnnymatthews — APPROVED** [review]
Removed the tree 🌲 emoji. It's cute, but it doesn't work well with translation services/extensions.

The rest of this looks good to me.

**Pull request merged**